src/ui/components/ExplorerHome/ExplorerHome.js

```diff
@@ -314,7 +393,10 @@ export const ExplorerHome = ({
               marginBottom: "20px",
             }}
           >
-            <span style={{fontSize: "24px"}}>Last Week&apos;s Activity</span>
+            <span style={{fontSize: "24px"}}>
+              {TIMEFRAME_OPTIONS[tab].tableLabel}
+            </span>
+            <span style={{fontSize: "16px"}}>{formatInterval(interval)}</span>
```

Comment:
Nit: whenever we touch font sizing styles, they should be converted to rem (or em) vs px in order to support accessibility: https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/

Reply:
accepted